code/game/gamemodes/endgame/supermatter_cascade/blob.dm

```diff
@@ -58,6 +61,9 @@
 					A = null
 				tcheck(80,1)
 			T.ChangeTurf(type)
+			var/turf/unsimulated/wall/supermatter/SM = T
+			if(SM.avail_dirs)
+				SM.avail_dirs -= get_dir(T, src)
```

Comment:
Now new supermatter tiles won't check spawn a new tile on the tile that spawned them